ballerina/Ballerina.toml

```
@@ -5,12 +5,12 @@ name = "hubspot.crm.commerce.quotes"
 version = "1.0.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = []
-# icon = "icon.png" # TODO: update icon.png
+keywords = [] # TODO: Add keywords
```

lets' add some short phrases that describe the package here. And remove the TODO
```suggestion
keywords = ["hubspot"] # TODO: Add keywords
```

---

done